staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -188,8 +189,19 @@ func (a customResourceStrategy) Validate(ctx context.Context, obj runtime.Object
 }
 
 // WarningsOnCreate returns warnings for the creation of the given object.
-func (customResourceStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
-	return nil
+func (a customResourceStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
+	var allWarnings []string
+	fldPath := field.NewPath("metadata")
+	objAccessor, err := meta.Accessor(obj)
```

Comment:
if we can't get an object accessor from the new object (or old object when old object is non-nil), just return an empty set of warnings